store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,51 @@ TODO: delete them permanently from here
+    <globalConfigValue>TRUE</globalConfigValue>
+    <desc>If true, enables Backup globally and if there is pre-configured data the crontab should get that schedule</desc>
+  </attr>
+<attr id="4009" name="zimbraBackupSyncUsers" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+    <globalConfigValue>FALSE</globalConfigValue>
+    <desc>If true,  will ensure the same users have backup and Dumpster enabled</desc>
+  </attr>
+<attr id="4010" name="zimbraBackupStartTime" type="astring" max="256" cardinality="single" optionalIn="globalConfig,server"  flags="serverInherited" since="9.1.0">
+    <globalConfigValue>02:00</globalConfigValue>
+    <desc>Starts backup each day at this time. This attribute is used for cronjob only to set hour</desc>
+  </attr>
+<attr id="4011" name="zimbraBackupRetentionDays" type="integer" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>8</globalConfigValue>
+    <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value</desc>
+  </attr>
+<attr id="4012" name="zimbraBackupBlobsCompress" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>TRUE</globalConfigValue>
+    <desc>If True, Blobs inside backups are compressed in zip format by default unless specified</desc>
+  </attr>
+<attr id="4013" name="zimbraBackupBlobsCompressZipStore" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>FALSE</globalConfigValue>
+    <desc>Backup up blobs in zip files without compression.</desc>
+  </attr>
+<attr id="4014" name="zimbraBackupSyncMode" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>FALSE</globalConfigValue>
+    <desc>Runs full backup synchronously</desc>
+  </attr>
+<attr id="4015" name="zimbraMailReportEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>TRUE</globalConfigValue>
+    <desc>If true, send an email report to the admin user.</desc>
+  </attr>
+<attr id="4016" name="zimbraBackupStandardFullBackup" type="astring" max="256" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>7</globalConfigValue>
+    <desc>Indicates a scheduled backup start day for full-backup</desc>
+  </attr>
+<attr id="4017" name="ZimbraBackupStandardIncBackup" type="astring" max="256" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>1</globalConfigValue>
+    <desc>Indicates a scheduled backup start day for incremental-backup which by default runs 6 out of 7 days except the day full-backup is triggered.</desc>
+  </attr>
```

Comment:
* same as zimbraBackupStandardFullBackup. What does `start day` mean? It is not easy to understand how the attr works.
* Is `astring` appropriate? `integer`?
* `Indicates` may be unnecessary.

Reply:
Removed this attribute. We are now only setting the zimbraBackupStartDay value from the UI which is used on the backend to set the full backup start day and incremental backup schedule for the other 6 days in a week for Standard backup.